scripts/toolboxSrc/primeIntegrations.ts

```diff
@@ -8,7 +8,7 @@ const upsertGlobalIntegrationProvidersFromEnv = async () => {
       authStrategy: 'oauth2',
       scope: 'global',
       teamId: 'aGhostTeam',
-      serverBaseUrl: 'https://gitlab.com',
+      serverBaseUrl: process.env.GITLAB_SERVER_URL ?? 'https://gitlab.com',
```

Comment:
Why a default value? IMHO we should just use the env GITLAB_SERVER_URL and on the .env.example we can have our default value.

Reply:
It's true that removing that default value is a breaking change. Ok, let's keep it there. 

I'm just against default hardcoded values in general 😸